scheduler/src/cook/mesos/api.clj

```diff
@@ -2584,7 +2583,7 @@
       (ANY "/running" []
         (running-jobs conn is-authorized-fn))
       (ANY "/list" []
-        (list-resource (db conn) framework-id is-authorized-fn)))
+        (list-resource (d/db conn) framework-id is-authorized-fn)))
```

Comment:
This would let you read list uncommitted jobs

Reply:
In this case, we should be OK. For job.state/waiting, we explicitly look at commit-latch. For running/completed, it doesn't matter.